frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx

```diff
@@ -138,6 +139,7 @@ export const CreateOBCPage: React.FC<CreateOBCPageProps> = (props) => {
                   {
                     isList: true,
                     kind: referenceForModel(NooBaaBucketClassModel),
+                    namespace: OCS_NS,
```

Comment:
Why not using `CEPH_STORAGE_NAMESPACE`  defined in ceph package.

Reply:
When we import something from Ceph into Noobaa, a dependency is created on ceph-plugin to be present before Noobaa can be used. I hope gradually we can move all the common shared stuff to console-shared and remove this dependency.

Reply:
Ceph is present as a dependency in noobaa package, we did that quite a while back.

Reply:
I am aware of that, or else breakdown card would not work. But wdyt, in the long run, wouldn't it be better is these two packages were less interdependent?

Reply:
I agree too the packages should be independent. Otherwise, we will phase the problem of circular dependency and we should move the code to a shared folder.

Reply:
@bipuladh its indeed a  good practice to have them independent, but the thing that noobaa is a part of OCS and will remain that in future, causes cross dependencies.

_A little bit history:_ When we wrote breakdown card in console shared, the console maintainers asked to move it back.We had no option (team acked on having ceph as a dependency for noobaa then), if somehow we can get their consensus on this then we are good to go. The knative and dev console plugin are another example for this.(knative imports stuff from dev console)

Also, we are sticking to the fact that always import from ceph and not reverse to avoid any circular dependencies.

PS: Plugins are static and they all will be loaded, we may encounter issues once they become dynamic (possible some day :/ )